metabase/metabase
# Add collections breadcrumbs and question lineage to mobile appbar

Epic https://github.com/metabase/metabase/issues/22826

Changes:
- Splits `AppBar` into `AppBarSmall` (mobile) and `AppBarLarge` (desktop) components
- Adds collection breadcrumbs and question lineage below the header for mobile
- Fixes the issue with collection breadcrumbs wrapping
- Fixes the issue with collection breadcrumbs only using 30% of the app bar width

How to test:
- Open a saved question and close the sidebar
- Resize the browser window and make sure the appbar works remains functional

<img width="500" alt="Screenshot 2022-06-28 at 22 37 29" src="https://user-images.githubusercontent.com/8542534/176270317-b538e5f5-226b-405c-9bf7-8a6fe1821c8c.png">
<img width="545" alt="Screenshot 2022-06-29 at 15 31 33" src="https://user-images.githubusercontent.com/8542534/176438891-b16fe479-0a37-43d9-bf0e-8a883c107078.png">
<img width="659" alt="Screenshot 2022-06-29 at 15 31 42" src="https://user-images.githubusercontent.com/8542534/176438911-cbab1f6c-b8d3-4e7a-bd65-8dc10782bef6.png">
<img width="1433" alt="Screenshot 2022-06-29 at 15 31 56" src="https://user-images.githubusercontent.com/8542534/176438927-5341ef26-2b30-412b-b90b-5a2d336e9b04.png">
